src/main/scala/scorex/core/network/SyncTracker.scala

```diff
@@ -38,7 +38,7 @@ class SyncTracker(nvsRef: ActorRef,
     schedule foreach {
       _.cancel()
     }
-    schedule = Some(context.system.scheduler.schedule(2.seconds, minInterval())(nvsRef ! SendLocalSyncInfo))
+    schedule = Some(context.system.scheduler.scheduleWithFixedDelay(2.seconds, minInterval(),nvsRef, SendLocalSyncInfo))
```

Comment:
better to beatify code on the way, e.g. 

//send SendLocalSyncInfo to NodeViewSynchronizer actor every minInterval()
val syncTask = context.system.scheduler.scheduleWithFixedDelay(2.seconds, minInterval(), nvsRef , SendLocalSyncInfo)
schedule = Some(syncTask)

Reply:
Done